scripts/install.sh

```diff
@@ -21,10 +21,9 @@ EOF
 RUN_COMPLETIONS=true
 
 parse_args() {
-  #BINDIR is ./bin unless set be ENV
-  # over-ridden by flag below
-
-  BINDIR=${BINDIR:-~/bin}
```

Comment:
Hi! Can you keep the current default location? It is already used on many machines and on its time it was decided to be set there after considering alternatives. ~/.local/bin is meant as a standard location on linux/mac, but there are many tools that just don't use it. Also, being hidden, it is not so easy to remember or locate.

Reply:
Not too sure about it being "easy or remember to locate", since `~/.local/bin` is the canonical location for `XDG_BIN_HOME`, meaning most OSes already have it in `$PATH` -- with `~/bin` (non-standard location seemingly made just for `avalanche-cli`, I have not seen a single tool that uses this), users have to *manually* add `~/bin` to their `$PATH`. If you still feel that `~/bin` is the best choice, I'll change it back, but I'd appreciate it if you would consider `~/.local/bin` one last time as to not break standard XDG convention.

Reply:
Yes please change it back. Sorry for the delay here.

Reply:
And thanks so much for the contribution and the feedback

Reply:
@felipemadero reverted it back to `~/bin`